src/ripple/app/tests/Path_test.cpp

```diff
@@ -0,0 +1,208 @@
+
+        auto master = createAccount("masterpassphrase", KeyType::ed25519);
+
+        Ledger::pointer LCL;
+        Ledger::pointer ledger;
+        std::tie(LCL, ledger) = createGenesisLedger(100000 * xrp, master);
+
+        // Create accounts
+        auto accounts = createAndFundAccountsWithFlags(master, { "alice", "bob", "mtgox" },
+            KeyType::ed25519, 10000 * xrp, ledger, LCL, asfDefaultRipple);
+        auto& alice = accounts["alice"];
+        auto& bob = accounts["bob"];
+        auto& mtgox = accounts["mtgox"];
+        expectNotEquals(alice.pk.humanAccountID(), master.pk.humanAccountID());
+
+        // Set credit limits
+        makeTrustSet(alice, mtgox, "USD", 600, ledger);
+        makeTrustSet(bob, mtgox, "USD", 700, ledger);
+
+        // Distribute funds.
+        makePayment(mtgox, alice, "USD", "70", ledger);
+
+        verifyBalance(ledger, alice, Amount(70, "USD", mtgox));
+
+        // Payment with path.
+        makePaymentWithPath(alice, bob, "USD", "24", ledger);
+
+        // Verify balances
+        verifyBalance(ledger, alice, Amount(46, "USD", mtgox));
+        verifyBalance(ledger, mtgox, Amount(-46, "USD", alice));
+        verifyBalance(ledger, mtgox, Amount(-24, "USD", bob));
+        verifyBalance(ledger, bob, Amount(24, "USD", mtgox));
+    }
+
+    void
+    test_path_find()
+    {
+        testcase("path find");
+
+        std::uint64_t const xrp = std::mega::num;
+
+        auto master = createAccount("masterpassphrase", KeyType::ed25519);
+
+        Ledger::pointer LCL;
+        Ledger::pointer ledger;
+        std::tie(LCL, ledger) = createGenesisLedger(100000 * xrp, master);
+
+        // Create accounts
+        auto accounts = createAndFundAccountsWithFlags(master, { "alice", "bob", "mtgox" },
+            KeyType::ed25519, 10000 * xrp, ledger, LCL, asfDefaultRipple);
+        auto& alice = accounts["alice"];
+        auto& bob = accounts["bob"];
+        auto& mtgox = accounts["mtgox"];
+        expectNotEquals(alice.pk.humanAccountID(), master.pk.humanAccountID());
+
+        // Set credit limits
+        makeTrustSet(alice, mtgox, "USD", 600, ledger);
+        makeTrustSet(bob, mtgox, "USD", 700, ledger);
+
+        // Distribute funds.
+        makePayment(mtgox, alice, "USD", "70", ledger);
+        makePayment(mtgox, bob, "USD", "50", ledger);
+
+        verifyBalance(ledger, alice, Amount(70, "USD", mtgox));
+        verifyBalance(ledger, bob, Amount(50, "USD", mtgox));
+
+        // Find path from alice to mtgox
+        auto alternatives = findPath(ledger, alice, bob, { Currency("USD") },
+            Amount(5, "USD", mtgox), log.stream());
+        log << "Path find alternatives: " << alternatives;
+        expectEquals(alternatives.size(), 1);
+        auto alt = alternatives[0u];
+        expectEquals(alt["paths_canonical"].size(), 0);
+        expectEquals(alt["paths_computed"].size(), 0);
+        /*
+        expectEquals(alt["paths_computed"].size(), 1);
```

Comment:
I will add a comment explaining that this is the old behavior. I kept it here because I'd like (me or someone else) to write a test that _does_ return data in the `paths_computed` field, and then this code can be moved/reused.